frontend/package.json

```diff
@@ -22,14 +23,15 @@
     "@oasisprotocol/dapp-voting-backend": "workspace:^",
     "@oasisprotocol/sapphire-paratime": "^1.3.2",
     "@phosphor-icons/core": "^2.0.8",
-    "ethers": "^6.11.1",
+    "ethers": "6.10.0",
```

Comment:
Nice! I would separate this into its own commit and note which dependency already loads this version of ethers (this deduplicates ethers dependency)

Reply:
Kinda mentioned this is due to `@oasisprotocol/sapphire-paratime` dependency in the PR desc. But sure, I can split up the commit.